src/DataFixtures/ORM/LoadUserData.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+namespace AppBundle\DataFixtures\ORM;
+
+use AppBundle\Entity\PostAddress;
+use AppBundle\Membership\AdherentFactory;
+use Doctrine\Common\DataFixtures\AbstractFixture;
+use Doctrine\Common\DataFixtures\FixtureInterface;
+use Doctrine\Common\Persistence\ObjectManager;
+use Symfony\Component\DependencyInjection\ContainerAwareInterface;
+use Symfony\Component\DependencyInjection\ContainerAwareTrait;
+use Tests\AppBundle\PropertyReplacer;
+
+class LoadUserData extends AbstractFixture implements FixtureInterface, ContainerAwareInterface
+{
+    use ContainerAwareTrait;
+
+    const USER_1_UUID = '313bd28f-efc8-57c9-8ab7-2106c8be9699';
+
+    public function load(ObjectManager $manager)
+    {
+        $adherentFactory = $this->getAdherentFactory();
+
+        // Create adherent users list
+        $user1 = $adherentFactory->createFromArray([
+            'uuid' => self::USER_1_UUID,
+            'password' => 'enmarche',
+            'email' => 'simple-user@example.ch',
+            'gender' => 'female',
+            'first_name' => 'Simple',
+            'last_name' => 'User',
+            'address' => PostAddress::createForeignAddress('CH', '8057', 'Zürich', '32 Zeppelinstrasse', 47.3950786, 8.5361402),
+            'birthdate' => '1972-11-24',
+        ]);
+
+        PropertyReplacer::replaceProperty($user1, 'adherent', false);
```

Comment:
Overkill, just add a parameters to the factory to disable the `join()` call in `createFromArray` method.

Reply:
Why? we should not break the encapsulation of the entity because of this line in fixtures loading.

Reply:
`\AppBundle\Membership\AdherentFactory::createFromArray` is only used for tests (which questionable too). So we can just add a new parameters to this method to conditionally called `$adherent->join();`. It's not breaking encapsulation.

Reply:
>is only used for tests (which is questionable)
Yup, like many factories, this is in my list of improvements ;).

Reply:
I fix it, but for me we can break the encapsulation for testing purpose.

Reply:
Sure if you have a very good reason 👍